### PR TITLE
Moving finance app to finance directory

### PR DESCRIPTION
**Pull request by dtest**
PR to move finance app to `finance` directory to make allowance for future sample apps in other verticals.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,157 +1,22 @@
...
 
-## Architecture & Schema
+More details of each sample application can be found in the README for the sample
+app.
 
-### Application Components
-This is how everything fits together:
-![FinAppConnect.png](images/FinAppConnect.png)
+Sample applications:
+- [Banking](finance/README.md)
```
Comment (c24t) [inline README.md:12]:
```suggestion
- [Finance](finance/README.md)
```

I know we called this "banking" in the original readme but let's go with "finance" unless we change it to "banking" everywhere, including in other docs and the sample app name.

File: finance/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Cloud Spanner Banking Application
```
Comment (c24t) [inline finance/README.md:1]:
```suggestion
# Cloud Spanner Finance Application
```

Not your change, I know, but this is a good time to update it.

File: README.md [hunk 3]
```diff
@@ -1,157 +1,22 @@
...
-We provide two alternative implementations for connecting to Cloud Spanner for
-educational purposes:
-- [Cloud Spanner Java Client](https://github.com/googleapis/java-spanner)
-- [Cloud Spanner JDBC Client](https://github.com/googleapis/java-spanner-jdbc)
 
-Clients, including the workload generator, a potential frontend component, or
-tools like [grpc_cli](https://github.com/grpc/grpc/blob/master/doc/command_line_tool.md)
-can connect to the FinAppServer using gRPC.
+## Code of Conduct
+Please review the [Code of Conduct](docs/code-of-conduct.md)
```
Comment (c24t) [inline README.md:16]:
```suggestion
Please review the [Code of Conduct](docs/code-of-conduct.md).
```

File: README.md [hunk 4]
```diff
@@ -1,157 +1,22 @@
...
+## Contributing
 
-The application models the following entities:
-- *Customer*: End-user of the application. A customer can have one or more
-  accounts.
-- *Account*: A financial account, keeps track of balance. Multiple customers
-  could access a single account with different roles.
-- *Transaction*: A monetary transfer associated with one or more accounts.
+If you are interested in contributing, more details on how to do so are provided 
+in the [Contributing page](docs/contributing.md).
```
Comment (c24t) [inline README.md:21]:
```suggestion
on the [Contributing page](docs/contributing.md).
```

Nit, feel free to ignore if you disagree.

**Review by c24t — APPROVED** [review]
LGTM with only minor comments, nice build fix and cleanup too.

File: README.md (hunk 3 above)
Comment (dtest) [reply README.md:16]: Added in #82

File: README.md (hunk 4 above)
Comment (dtest) [reply README.md:21]: Added in #82

File: finance/README.md (hunk 2 above)
Comment (dtest) [reply finance/README.md:1]: Added in #82

File: README.md (hunk 1 above)
Comment (dtest) [reply README.md:12]: Added in #82

**Pull request merged**